packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -145,6 +148,15 @@ class CalendarDatePicker extends StatefulWidget {
   /// Function to provide full control over which dates in the calendar can be selected.
   final SelectableDayPredicate? selectableDayPredicate;
 
+  /// The box decoration of selected day.
+  final BoxDecoration? selectedDayDecoration;
+
+  /// The box decoration of disabled day.
+  final BoxDecoration? disabledDayDecoration;
+
+  /// The box decoration of today.
+  final BoxDecoration? todayDecoration;
+
```

Comment:
Can these be `Decoration` instead of `BoxDecoration` to make them more general?

Reply:
Done👍